crates/rpc/rpc-types/src/lib.rs

```diff
@@ -55,3 +55,17 @@ pub use eth::{
 
 pub use peer::*;
 pub use rpc::*;
+
+use alloy_eips::eip4844::BYTES_PER_BLOB;
+use alloy_primitives::FixedBytes;
+use serde::{Deserialize, Serialize};
+
+/// Blob type returned in responses to `engine_getBlobsV1`.
+// FIXME(sproul): move to alloy?
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct BlobAndProofV1 {
+    /// The blob data.
+    pub blob: FixedBytes<BYTES_PER_BLOB>,
```

Comment:
I think we should definitely box this because we do `vec![None; request.len()]` so each entry even if missing will be a full blob